search_events_app/tests.py

```diff
@@ -9,50 +9,56 @@
 from search_events_app.models.feature import Feature
 
 
-class TestModels(TestCase):
+class TestCountry(TestCase):
+
+    def setUp(self):
+        self.country = Country.objects.create(name="Argentina", alpha_2_code="AR",
+        alpha_3_code="ARG", flag="https://restcountries.eu/data/arg.svg")
+
+    def test_country_basic_info(self):
+        self.assertEqual(self.country.name, "Argentina")
+        self.assertEqual(self.country.alpha_2_code, "AR")
+        self.assertEqual(self.country.alpha_3_code, "ARG")
+        self.assertEqual(self.country.flag, "https://restcountries.eu/data/arg.svg")
+
+    def test_country_str(self):
+        self.assertEqual(self.country.__str__(), "Argentina")
+
+    
+class TestEvent(TestCase):
+
     def setUp(self):
         self.country = Country.objects.create(name="Argentina", alpha_2_code="AR",
         alpha_3_code="ARG", flag="https://restcountries.eu/data/arg.svg")
+        self.event2 = Event("Name", "www.google.com")
         self.feature = Feature("Embedded Checkout")
-        self.feature2 = Feature("Reserved Seating")
         self.event = Event(
             "Evento1", "www.google" +
             ".com", self.country, self.feature
-            )
-        self.event2 = Event("Name", "www.google.com")
+        )
+        self.feature2 = Feature("Reserved Seating")        
```

Comment:
Same as above. if self.feature2 is only used by self.new_event and self.new_event is only used in one method, you should move this to the specific method...
This makes the tests more readable and also if you change something related to this property in the future, tests that not use this property will fail and maybe cause you more confusion.